functions/src/repository/response-git-webhook-repository.ts

```diff
@@ -145,9 +146,21 @@ async function repositoryEvent(data: RepositoryEventModel): Promise<void> {
 async function pullRequestEvent(data: PullRequestEventModel): Promise<void> {
   Logger.info('pullRequestEvent');
   const repository: DocumentData = await RepositoryModel.getRepositoryById(data.repository.id);
-
   data.updateData(repository);
 
+  // Find logged in user to get the github token
+  const usersRef: QuerySnapshot = await (FirebaseAdmin.firestore().collection('users').where('username', '==', data.sender.login).get());
+
+  if (!usersRef.empty) {
```

Comment:
Every pull request will have a user, why is this check needed?

Reply:
I will remove it .